### DISCUSSION TITLE
Spotify - change Accountname

### DISCUSSION THREAD

**Discussion by Borlizgaron** [MiczFlor/RPi-Jukebox-RFID#1617] [Q&A] [answered]
Hey everybody
the Phoniebox is a great piece of work. My kids love it, I love it.
Recently I reinstalled the box and must have entered my Spotify account name wrong. Instead of the number combination, I chose the email address. 
Is it enough to put the correct data in the mopidy.conf or do I have to reinstall everything?

**Comment by MiczFlor** [comment, marked as answer]
Hi @Borlizgaron 
have you tried simply changing the account name in `mopidy.conf`?
When searching through the code for `%spotify_username%`, I came to find these two locations that the installation seems to work with:
* /etc/mopidy/mopidy.conf
* ~/.config/mopidy/mopidy.conf
You might need to change the user credentials in both files.
Hope this helps, all the best, micz

**Comment by Borlizgaron** [comment]
Hey @MiczFlor ,
unfortunately I could not change anything there. I have no access rights as root, only "mopidy" can access it.
I reinstalled everything. After that it worked.
Now I'm struggling with other problems, but one step at a time. :D
